pages/dkp/kommander/2.2/dkp-upgrade/index.md

```diff
@@ -1,10 +1,62 @@
+
+## Understand the upgrade process
+
+For this release, you perform the upgrade sequentially beginning with DKP Kommander and then moving to DKP Konvoy.
+
+When upgrading DKP, the process is different depending on whether you run a stand-alone Management Cluster, or a multi-cluster environment that includes a combination of a Management Cluster and Workspace Clusters. This can include both attached and managed clusters. 
+
+Start with your Management Cluster in Kommander, and then proceed, Workspace by Workspace, if more than one exists. Then you can move to upgrading Konvoy, cluster by cluster.
+
+The overall process for upgrading to the latest version of DKP is done on each Workspace or cluster, and has the following high-level steps.
+
+   For **Kommander**, on your Management Cluster:
+
+1.  [**Upgrade Kommander**][upgrade_kommander], which upgrades all Platform Applications.
+
+    If you do not have any Managed or Attached Clusters, skip to upgrading Konvoy on your Management Cluster.
+    
+
+   For **Kommander**, on your Workspaces (which include Management and Managed clusters):
+
+1.  [**Upgrade Kommander on your Workspaces**][upgrade_workspaces], which upgrades all Platform Applications on your Workspace Clusters (Workspace Clusters can include Attached and Managed Clusters).
+
+1.  [**Upgrade all DKP Catalog applications**][catalog_apps] deployed to any Workspaces or Projects.
+
+1.  [**Verify any Custom Catalog applications**][custom_apps] and ensure they are compatible with the Kubernetes version included in the [new release][release_notes].
+
+
+   For **Konvoy**, on your Management Cluster:
+
+1.  [**Upgrade Konvoy’s CAPI components**][upgrade_konvoy_capi]. This upgrades the CAPI controllers, which only run on the Management Cluster.
+
+1.  [**Upgrade the Core Addons**][upgrade_konvoy_core]. This upgrades multiple addons such as CSI, CNI, Cluster Autoscaler, and Node Feature Discovery.
+
+1.  [**Upgrade the Kubernetes version**][upgrade_konvoy_kubernetes]. This upgrades your cluster’s control plane and node pools.
+
+    If you do not have any Managed or Attached Clusters, you have finished the upgrade process and can start testing your environment. If you have Managed Clusters, continue with the next section.
+    
+
+   For **Konvoy**, on your Managed Clusters:
+
+1.  [**Upgrade the Core Addons**][upgrade_konvoy_core]. This upgrades multiple addons such as CSI, CNI, Cluster Autoscaler, and Node Feature Discovery.
+
+1.  [**Upgrade the Kubernetes version**][upgrade_konvoy_kubernetes]. This upgrades your cluster’s control plane and node pools. You may also want to upgrade your Kubernetes version on any Attached clusters.
+
```

Comment:
"May also want to..." - please check with an Eng SME on this. I'm thinking this might be recommended behavior, in which case we should say "We recommend you upgrade..." here. It's a stronger statement than "you might want to", but not as strong as "you must".